examples/fluid/falling_water_spheres_2d.jl

```diff
@@ -90,6 +90,6 @@ callbacks = CallbackSet(info_callback, saving_callback)
 
 # Use a Runge-Kutta method with automatic (error based) time step size control.
 sol = solve(ode, RDPK3SpFSAL35(),
-            abstol=1e-6, # Default abstol is 1e-6
+            abstol=1e-7, # Default abstol is 1e-6
```

Comment:
why do you need to change this?
What do the falling spheres have to do with the viscosity? I'm curious

Reply:
This case uses two fluid systems which interact.

Reply:
It also uses adami and Monaghan viscosity at the same time. So the solution changes.

Reply:
Ah sorry I confused it with the falling spheres example